### PR TITLE
Support WebGPU

### PR DESCRIPTION
**Pull request by DanielHauschildt**
Added support for

- WebGPU 
- Patched ISNET for WebGPU execution
- Onnx runtime with WebGPU support
- Smaller ModNet Model

Added "device" parameter with options "cpu | gpu". For now "cpu" is default.

### CODE REVIEW

File: packages/web/src/onnx.ts [hunk 1]
```diff
@@ -1,47 +1,70 @@
...
-      !capabilities.simd && capabilities.threads
+      !useWebGPU && useThreads && !useSimd
         ? await loadAsUrl('/onnxruntime-web/ort-wasm-threaded.wasm', config)
         : undefined,
     'ort-wasm.wasm':
-      !capabilities.simd && !capabilities.threads
+      !useWebGPU && !useThreads && !useSimd
         ? await loadAsUrl('/onnxruntime-web/ort-wasm.wasm', config)
         : undefined
   };
```
Comment (mirko314) [inline packages/web/src/onnx.ts:70]: I find this code path seems a bit hard to scan, I'd personally like to simplify this logic

File: packages/web/changelog/1.4.5/20242602190400-Added_ThirdPartyLicenses_json_Added.yaml [hunk 2]
```diff
@@ -6,4 +6,4 @@ type: Added
 # type: Security
 # private: true
 description: |
-    Added ThirdPartyLicenses.json Added
+  Added ThirdPartyLicenses.json Added
```
Comment (mirko314) [inline packages/web/changelog/1.4.5/20242602190400-Added_ThirdPartyLicenses_json_Added.yaml]:
```suggestion
  Added ThirdPartyLicenses.json 
```

**Review by mirko314 — CHANGES_REQUESTED** [review]
Works great and super quick in Chrome!!
However, I could not get the example to work in Safari and Firefox. I'd expect a gracefull fallback to lower-performance runtimes until one is found that works in the browser

File: packages/web/src/onnx.ts (hunk 1 above)
Comment (DanielHauschildt) [reply packages/web/src/onnx.ts:70]: Feeded your comment to CoPilot and my sidekick wants to change it to this: 
  
  
  const wasmPaths = {
    'ort-wasm-simd-threaded.wasm': useThreads && useSimd,
    'ort-wasm-simd.wasm': !useThreads && useSimd,
    'ort-wasm-threaded.wasm': !useWebGPU && useThreads && !useSimd,
    'ort-wasm.wasm': !useWebGPU && !useThreads && !useSimd
  };

  for (const [key, value] of Object.entries(wasmPaths)) {
    if (value) {
      const wasmPath = useWebGPU && key.includes('simd') ? `/onnxruntime-web/${key.replace('.wasm', '.jsep.wasm')}` : `/onnxruntime-web/${key}`;
      ort.env.wasm.wasmPaths[key] = await loadAsUrl(wasmPath, config);
    }
  }

**Pull request merged**